libnavui-dropin/src/main/res/values-night/colors.xml

```diff
@@ -1,6 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="mapbox_button_fg_color">@color/colorOnSurface</color>
-    <color name="mapbox_button_bg_color">@color/colorSurface</color>
+    <color name="mapbox_drop_in_primary">@color/colorSecondary</color>
+    <color name="mapbox_drop_in_surface">#fff</color>
```

Comment:
Let us not define any new colors here and use from `libnavui-resources/res/colors.xml`